src/packages/@ncigdc/utils/analytics/performanceTracker.js

```diff
@@ -0,0 +1,34 @@
+const startTimes = {};
+export const timer = {
+  time: label => (startTimes[label] = performance.now()),
+  timeEnd: label => {
+    if (!startTimes[label]) {
+      console.warn(`No start time was found for "${label}"`);
+      return;
+    }
+    const duration = performance.now() - startTimes[label];
```

Comment:
should probably remove startTimes here, to prevent calling timeEnd multiple times.